File: Backend/src/main/java/com/example/backend/models/StorageProduct.java
package com.example.backend.models;
import lombok.Data;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
@Data
@Setter
@Getter
public class StorageProduct {
    private String id;
    private String name;
    private String brand;
    private double weight;
    private Date expirationDate;
    private Measure measure;
}

File: Mobile/ProjectBogdan-main/app/src/main/java/com/example/projectbogdan/data/model/StorageProduct.kt
package com.example.projectbogdan.data.model

import java.util.*

data class StorageProduct(
    val id: String = "",
    val name: String = "",
    val brand: String = "",
    var weight: Double = 0.0,
    var expirationDate: Date? = null,
    var measure: String = ""
)